router/src/validation.rs

```diff
@@ -279,6 +293,11 @@ impl Validation {
             do_sample,
             seed,
             watermark,
+            use_grammar_constraint,
+            grammar,
+            logit_bias: logit_biases,
```

Comment:
if the names was changed to `logit_bias`, then we can simplify this a bit
```suggestion
            logit_bias,
```